docs/source/upgrade-guide/index.md

```diff
@@ -1684,7 +1685,7 @@ Add this key to the `jest.moduleNameMapper`:
   }
 ```
 
-because the new version of Jest is a bit more picky when importing externals. Attention, this mapping needs to be the first, it needs to come before the `@plone/volto/(.*)$` key.
+because the new version of Jest is a bit more picky when importing externals. Attention, this mapping needs to be the first, it needs to come before the ``@plone/volto/(.*)$`` key.
```

Comment:
Revert.

```suggestion
because the new version of Jest is a bit more picky when importing externals. Attention, this mapping needs to be the first, it needs to come before the `@plone/volto/(.*)$` key.
```